files/image_config/sudoers/sudoers

```diff
@@ -34,6 +34,7 @@ Cmnd_Alias      READ_ONLY_CMDS = /bin/cat /var/log/syslog*, \
                                  /usr/local/bin/decode-syseeprom, \
                                  /usr/local/bin/generate_dump, \
                                  /usr/local/bin/lldpshow, \
+                                 /usr/local/bin/pcieutil *, \
```

Comment:
Please add testcases in https://github.com/Azure/sonic-mgmt/blob/master/tests/tacacs/test_ro_user.py#L44

Reply:
Draft PR here: https://github.com/Azure/sonic-mgmt/pull/2930